### PR TITLE
Removes tactical suicide borging

### LINKED ISSUE #39796: Suicide is faster for debraining
Instead of doing the operation, roboticists ask their patients to kill themselves with a box to debrain them using the suicide verb. Not a bug, but it doesn't seem like intended gameplay

### LINKED ISSUE #40753: Double Esword causes Out-of-VR death when used for suicide in VR
Issue reported from  Round ID: 95044 (/tg/Station Sybil [ENGLISH] [US-EAST] [100% LAG FREE])

Reporting client version: 512

Sorry if this issue is kinda bad. I normally don't report bugs, but i found this one out

[Reproduction]; When you are in VR, suiciding with double esword wont boot you from vr, but will kill you and make you a ghost. I have done it multiple times now, and doesn't seem to be a one-time occurrence. I have not tried it  with other items with custom suicides, but it may have the same problem. Ill comment on this issue if i find more items that cause out-of-vr death when used in VR.

### PR DESCRIPTION
**Pull request by ShizCalev**
fixes #39796
fixes #40753

:cl: ShizCalev
fix: Due to a recent policy review by Nanotrasen Central Command, we have determined that personnel who have committed suicide are too great of a risk to be borged, and are making half of the Robotics staff's job redundant (we pay them good money to do that!) As such, suicide is now considered a PERMANENT decision.
fix: Fixed a specific scenario where ghosts would not be passed their suicider status.
fix: Fixed a specific scenario where brains that suicided would not have their suicider var corrected upon being revived.
fix: Defibrillator units will now correctly consider 180 damage in either brute or burn damage to be the cutoff for revival (down from 181.)
fix: Fixed mobs suiciding in VR.
/:cl:

shockpaddles/can_defib() is now human readable (serious please stop with the 5000 character long chains if it's not needed)

### CODE REVIEW

**Comment by Barhandar** [comment]
On one hand it was stupid that you could be revived from suicide by brain cloning (and brain-removing suicides), on the other brain removal via regular means (organ extraction, or head amputation + headbutcher) is way too slow even for Sybil. Can you do organ harvesting on live subjects?

**Comment by Dorsisdwarf** [comment]
I think this is a dumb PR but if "head chopped off with a saw then saw it again" is too slow for you you may want to ease it on the adderall

**Comment by DaxDupont** [comment]
Aww this was so cool to do

File: code/game/objects/items/defib.dm [hunk 1]
```diff
@@ -437,8 +437,18 @@
...
+	if(H.suiciding || H.has_trait(TRAIT_NOCLONE) || H.hellbound)
+		return
+	if((world.time - H.timeofdeath) > tlimit)
+		return
+	if((H.getBruteLoss() >= MAX_REVIVE_BRUTE_DAMAGE) && (H.getFireLoss() >= MAX_REVIVE_FIRE_DAMAGE))
+		return
+	if(!H.getorgan(/obj/item/organ/heart))
+		return
 	var/obj/item/organ/brain/BR = H.getorgan(/obj/item/organ/brain)
-	return	(!H.suiciding && !(H.has_trait(TRAIT_NOCLONE)) && !H.hellbound && ((world.time - H.timeofdeath) < tlimit) && (H.getBruteLoss() < 180) && (H.getFireLoss() < 180) && H.getorgan(/obj/item/organ/heart) && BR && !BR.damaged_brain)
```
Comment (PKPenguin321) [inline code/game/objects/items/defib.dm]: 🤢 🤧

**Comment by ShizCalev** [comment]
It's cool that you could do it, but it's not the intended purpose of suiciding.

Suiciding shouldn't be a "quick borging" verb. Per the wiki;
image

**Comment by evsey9** [comment]
if it's an ooc action, make it fail the "die a glorious death" traitor objective too. suicide without means of revival is not glorious.

**Comment by evsey9** [comment]
thanks

**Comment by Arianya** [comment]
Long overdue really. :+1:

**Comment by Time-Green** [comment]
I know it should be fixed, but im gonna miss blowing my brain right out of my head and into the roboticists hands with a foam pen shotgun

**Comment by octareenroon91** [comment]
>organ harvester

Please give science access to fabricate this.

**Comment by PKPenguin321** [comment]
No, that PR was closed by its author

**Pull request merged**